x-pack/plugins/cloud_security_posture/public/components/cloud_posture_page.tsx

```diff
@@ -65,12 +64,18 @@ const packageNotInstalledRenderer = (cisIntegrationLink?: string) => (
           title: i18n.translate('xpack.csp.cloudPosturePage.packageNotInstalled.buttonLabel', {
             defaultMessage: 'Add a CIS integration',
           }),
-          description: i18n.translate(
-            'xpack.csp.cloudPosturePage.packageNotInstalled.description',
-            {
-              defaultMessage:
-                'Use our CIS Kubernetes Benchmark integration to measure your Kubernetes cluster setup against the CIS recommendations.',
-            }
+          description: (
+            <FormattedMessage
+              id="xpack.csp.cloudPosturePage.packageNotInstalled.description"
+              defaultMessage="Use our {integrationFullName} (KSPM) integration to measure your Kubernetes cluster setup against CIS recommendations."
```

Comment:
we're using this link twice, once in `learn more` and the other in `use our <link> KSPM integration ...` 

maybe we can just use the first one